documentation/modules/common-attributes.adoc

```diff
@@ -8,7 +8,7 @@
 :ocp: Red Hat OpenShift
 :ocp-name: OpenShift
 :ocp-short: OpenShift
-:ocp-version: 4.11
+:ocp-version: 4.15
```

Comment:
I see that `ocp-version` is used mainly in links to the OpenShift documentation but also in the prerequisites of the MTV operator: "* {ocp} {ocp-version} or later installed." - and in that case, it should be the earliest supported version, which in case of 2.6 is 4.14

Reply:
Good thinking. Fixed.